src/main/java/use_case/cat_management/change_cat_happiness/ChangeCatHappinessInteractor.java

```diff
@@ -19,8 +19,9 @@ public ChangeCatHappinessInteractor(CatDataAccessInterface catDataAccessObject,
 
     @Override
     public void execute(ChangeCatHappinessInputData changeCatHappinessInputData) {
-        // get the cat, cat object should already exist dues to create cat use case
-        final Cat cat = changeCatHappinessInputData.getCat();
+        // get the cat, cat should already exist
+        final Cat cat = catDataAccessObject.getCatByNameAndOwner(changeCatHappinessInputData.getCatName(),
+                changeCatHappinessInputData.getOwnerUsername());
         int newHappiness = 0;
 
         // decrease happiness when user does not complete study session
```

Comment:
Fix return count of calculateHappinessPoints and magic numbers (Checkstyle errors)